src/js/components/Page/index.d.ts

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+import { BoxProps } from '../Box/index';
+
+export interface PageProps {
+  contentProps?: BoxProps;
+  customizeContent?: boolean;
```

Comment:
Agreed on wanting a better name for this. One idea:
- `plain` - since you could put any children you want and they would fill the Page width, aligns with Button, default false